layouts/partials/footer.hbs

```diff
@@ -29,16 +29,18 @@
 
 <link rel="stylesheet" href="/static/css/prism-tomorrow.css" media="all">
 <script>
-  var $scrollToTop = document.getElementById('scroll-to-top');
+  var scrollToTop = document.getElementById('scroll-to-top');
+
   (window.onscroll = function() {
```

Comment:
If I change this to use `addEventListener` then the link shows on page load. Maybe this can be improved further, so any ideas welcome.